modules/calibration/data/ch/calibration_table.pb.txt

```diff
@@ -0,0 +1,7920 @@
+control_period: 0.01
+trajectory_period: 0.1
+chassis_period: 0.01
+localization_period: 0.01
+max_status_interval_sec: 0.1
+max_planning_interval_sec: 0.2
+max_planning_delay_threshold: 4.0
+action: STOP
+soft_estop_brake: 0.0
+active_controllers: LON_CONTROLLER
+active_controllers: LAT_CONTROLLER
+max_steering_percentage_allowed: 100
+minimum_speed_resolution: 0.2
+query_relative_time: 0.8
+minimum_speed_protection: 0.1
+lat_controller_conf {
+  ts: 0.01
+  preview_window: 0
+  cf: 155493.663
+  cr: 155493.663
+  mass_fl: 55
+  mass_fr: 55
+  mass_rl: 65
+  mass_rr: 65
+  eps: 0.01
+  matrix_q: 0.05
+  matrix_q: 0.0
+  matrix_q: 1.0
+  matrix_q: 0.0
+  cutoff_freq: 10
+  mean_filter_window_size: 10
+  max_iteration: 150
+  max_lateral_acceleration: 5.0
+  lat_err_gain_scheduler {
+    scheduler {
+      speed: 2.0
+      ratio: 1.0
+    }
+    scheduler {
+      speed: 3.0
+      ratio: 0.6
+    }
+    scheduler {
+      speed: 4.0
+      ratio: 0.2
+    }
+    scheduler {
+      speed: 5.0
+      ratio: 0.1
+    }
+    scheduler {
+      speed: 6.0
+      ratio: 0.05
+    }
+  }
+  heading_err_gain_scheduler {
+    scheduler {
+      speed: 4.0
+      ratio: 1.0
+    }
+    scheduler {
+      speed: 8.0
+      ratio: 0.6
+    }
+    scheduler {
+      speed: 12.0
+      ratio: 0.4
+    }
+    scheduler {
+      speed: 20.0
+      ratio: 0.2
+    }
+    scheduler {
+      speed: 25.0
+      ratio: 0.1
+    }
+  }
+}
+lon_controller_conf {
+  ts: 0.01
+  brake_deadzone: 10
+  throttle_deadzone: 10.0
```

Comment:
`brake_deadzone` `throttle_deadzone` these parameters have moved to `vehicle_param.pb.txt`,  you need a test and configuration field clean on master branch.

Reply:
I have already modified them, please review again.